anonymizer/base.py

```diff
@@ -317,7 +328,7 @@ def run(self):
                 values[obj.pk] = updates
 
                 index += 1
-                if index % step_size == 0:
+                if index % step_size == 0 or index == count:
```

Comment:
this was a bug that has existed for a while, the last chunk/step wasn't being anonymized